instrumentation/vertx/vertx-sql-client-4.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/vertx/v4_0/sql/VertxSqlClientRequest.java

```diff
@@ -16,10 +16,18 @@ public VertxSqlClientRequest(String statement, SqlConnectOptions sqlConnectOptio
     this.sqlConnectOptions = sqlConnectOptions;
   }
 
+  /**
+   * @deprecated Use {@link #getDbQueryText()} instead.
+   */
+  @Deprecated
   public String getStatement() {
     return statement;
   }
```

Comment:
this class is only used in the Java agent, so this method can be deleted (doesn't need to go through deprecation cycle)